src/dispatcher.ts

```diff
@@ -14,8 +14,9 @@ export function subscribe(actionId: string, callback: Subscriber<any>) {
 }
 
 export function dispatch(actionMessage: ActionMessage) {
-    if (getGlobalContext().inMutator) {
-        throw new Error('Mutators cannot dispatch further actions.');
+    const currentMutator = getGlobalContext().currentMutator;
+    if (currentMutator) {
+        throw new Error(`Action(${actionMessage.type}) was call inside mutator(${currentMutator})`);
```

Comment:
>`Action(${actionMessage.type}) was call inside mutator(${currentMutator})` [](start = 24, length = 74)

I want to make sure this message isn't just a "this happened" message but a "this is not allowed" message.  Can you change it to something like: `Mutator (foo) may not dispatch action (bar)`